internal/lint/src/main/java/com/firebaseui/lint/LintIssueRegistry.kt

```diff
@@ -6,5 +6,5 @@ import com.android.tools.lint.client.api.IssueRegistry
  * Registry for custom FirebaseUI lint checks.
  */
 class LintIssueRegistry : IssueRegistry() {
-    override fun getIssues() = listOf(NonGlobalIdDetector.NON_GLOBAL_ID)
+    override val issues = listOf(NonGlobalIdDetector.NON_GLOBAL_ID)
```

Comment:
`override val` --> Kotlin is cool but also sometimes it's stupid.  This is not an actionable comment, just being grumpy Java guy.

Reply:
Awww, c'mon!? How can you _not_ love overriding fields? It's so clean! ✨